data/hai/hai reveal 6 end.txt

```diff
@@ -32,10 +32,10 @@ mission "Hai Reveal [C01] Cleanup"
 			`When you land, you, Teeneep, and the Unfettered Hai are greeted as heroes. Word has traveled ahead of you through the new message links, and you are greeted by a hastily planned celebration in the capitol. After a few hours involving a brief formal parade, a great deal of cheering, some minor speeches with Xilin and Alondo trotted out to reassure everyone that there would be no conflict, and finally some more discrete parties outside the public eye, General Choot'k comes to you and Teeneep to bid his farewell.`
 			`	He shakes your hand and says, "You're a fine warrior, Captain <last>. You are as much a True Hai as any human can be, and it was an honor to fight alongside you in battle."`
 			`	Then he turns to Teeneep and tells her, "You're the future leader of all our people, Teeneep. Come back with me to Darkcloak. Join the True Hai and lead us to victory."`
-			`	Teeneep smiles and says, "My fight is here general. I fight for the hearts and minds of our Fettered brethren that we may find a true path. One day again we will be one people."`
+			`	Teeneep smiles and says, "My fight is here, general. I fight for the hearts and minds of our Fettered brethren that we may find a true path. One day we will again be one people."`
```

Comment:
The word order here could be intentional (although it would probably need more commas to work). The added comma is definitely a good catch, though.